src/hotspot/share/jvmci/jvmciRuntime.cpp

```diff
@@ -267,6 +267,20 @@ JRT_ENTRY_NO_ASYNC(static address, exception_handler_for_pc_helper(JavaThread* t
   }
 #endif
 
+  // debugging support
+  // tracing
+  if (log_is_enabled(Info, exceptions)) {
+    ResourceMark rm;
+    stringStream tempst;
+    assert(cm->method() != NULL, "Unexpected null method()");
+    tempst.print("compiled method <%s>\n"
+                 " at PC" INTPTR_FORMAT " for thread " INTPTR_FORMAT,
+                 cm->method()->print_value_string(), p2i(pc), p2i(thread));
+    Exceptions::log_exception(exception, tempst.as_string());
+  }
+  // for AbortVMOnException flag
+  NOT_PRODUCT(Exceptions::debug_check_abort(exception));
```

Comment:
All other places calls debug_check_abort() in product too. It seems 8136577 changes missed JVMCI:
https://github.com/openjdk/jdk/commit/a25ce80e782a73390ae1109250dfa6fe6a48fcbe
Please, update to call it in product for JVMCI too. Note, AbortVMOnException is diagnostic flag now.